docs/usage/validators.md

```diff
@@ -195,7 +217,43 @@ print(DemoModel(int_list=[3, 2, 1], name_list=['adrian g', 'David']))
 #> int_list=[1, 2, 3] name_list=['Adrian G', 'David']
 ```
 
-## Validate Always
+### Wrap validators
+
+Wrap validators are useful for cases where you want to validate a value, but also want to return a default value if validation fails.
+
+```py
```

Comment:
Did this come from a test? If so we can delete the test

Reply:
No this is adapted from the docstring